docs/standard-library/built-in-decorators.md

```diff
@@ -613,6 +697,60 @@ scalar distance is float64;
 ```
 
 
+### `@nextLink` {#@nextLink}
```

Comment:
to me a `@nextLink` is a form of `continuationToken`, it almost seems more "azure" than unbranded. is this expressed anywhere? would it be better to think of a continuation token as something that can be a link / query / header param, where a continuation token link is a `nextLink`

Reply:
it is quite common(e.g. github api), defining them in rel header is a common use case

Reply:
and its not necessarly tied to a contiuation token, the next link might still use `?page=4` to indicate the next page

Reply:
that's a good point